DDG4/python/bin/g4GraphicalScan.py

```diff
@@ -0,0 +1,381 @@
+###############################################################
```

Comment:
```suggestion
#!/usr/bin/env python
###############################################################
```

Reply:
Should this be `python3`?

Reply:
I copied this line from two other files in this folder.

Reply:
Right, then this might be some legacy? Canonically it should be `python3` as that is the name of the executable, but I think on many platforms `pyhton` is symlinked to `python3`, so that it might not matter as much.